docs/content/guides/dagster/transitioning-data-pipelines-from-development-to-production.mdx

```diff
@@ -265,22 +343,29 @@ resources = {
 
 ## Advanced: Unit tests with stubs and mocks
 
-You may have noticed a missing step in the development workflow presented in this guide — unit tests! While the main purpose of the guide is to help you transition your code from local development to a production deployment, unit testing is still an important part of the development cycle. In this section, we'll explore a pattern you may find useful when writing your own unit tests.
+You may have noticed a missing step in the development workflow presented in this guide — unit tests! While the main purpose of this guide is to help you transition your code from local to production deployment, unit testing is still an important part of the development cycle. In this section, we'll explore a pattern you may find useful when writing your own unit tests.
```

Comment:
With the change I propose in [1] I think we should change the terminology in this section from unit tests to tests. it's a bit pedantic, but since the snippet i wrote does read/write data to snowflake, it's not quite lightweight enough to be considered a unit test

Reply:
Fine by me!